accelerator/apis.c

```diff
@@ -25,6 +28,20 @@ int apis_logger_release() {
   return 0;
 }
 
+status_t apis_lock_init() {
+  if (pthread_mutex_init(&cjson_lock, NULL)) {
```

Comment:
would it better to use the the lock and thread in entangled in `entangled/utils/handles`?

Reply:
Great point! I'll modify it